src/content/blog/2024_qcoder.md

```diff
@@ -156,6 +156,6 @@ You now have access to all the content on QCoder. We suggest starting by partici
 
 # Conclusion
 [QCoder](https://www.qcoder.jp/) is a quantum competitive programming platform, offering opportunities to challenge and advance your skills in quantum computing.
-We are excited to announce that our 3rd contest is scheduled for November (date TBD). We look forward to your participation!
+We are excited to announce that our 3rd contest, [QCoder Programming Contest 003](https://www.qcoder.jp/contests/QPC003) (QPC003), will take place on **November 3rd, from 8:00 AM to 11:00 AM (GMT+0)**. We look forward to your participation!
```

Comment:
I have updated the upcoming contest schedule with the contest link (the page will be available soon). I'm ready to publish!